### PR TITLE
add zipkin to consul and templatize, update consul bookinfo images

### PR DESCRIPTION
**Pull request by rshriram**
**What this PR does / why we need it**:
templatize consul install file so that the Pilot version is automatically updated during release. Also, update bookinfo images with latest proxy-debug, and add Zipkin to Consul install (tested and works very well).

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
Needed before 0.2.7 release.
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: samples/bookinfo/consul/bookinfo.yaml [hunk 1]
```diff
@@ -18,15 +19,16 @@ services:
...
     networks:
       istiomesh:
     dns:
       - 172.28.0.1
       - 8.8.8.8
     dns_search:
         - service.consul
-    privileged: true
+    cap_add:
+      - NET_ADMIN
```
Comment (rshriram) [inline samples/bookinfo/consul/bookinfo.yaml]: fyi @GregHanson and @linsun - tested with cap net_Admin and it works. So no need to give elevated "priviledged:true".

File: samples/bookinfo/src/details/start_service.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 
 /opt/istio/prepare_proxy.sh -p 15001 -u 1337
 ruby /opt/microservices/details.rb 9080 &
-su istio-proxy -c "/opt/istio/pilot-agent proxy -v 2 --serviceregistry Consul > /tmp/envoy.log"
+su istio-proxy -c "/opt/istio/pilot-agent proxy -v 2 --serviceregistry Consul --serviceCluster details-v1 --zipkinAddress zipkin:9411  > /tmp/envoy.log"
```
Comment (rshriram) [inline samples/bookinfo/src/details/start_service.sh]: This update allows sending traces to Zipkin.. Also, we get a nice service graph from zipkin.

File: install/consul/istio.yaml [hunk 3]
```diff
@@ -36,7 +38,7 @@ services:
           - consul
     ports:
       - "8500:8500"
-      - "172.28.0.1:53:8600/udp"
+      - "53:8600/udp"
       - "8400:8400"
```
Comment (linsun) [inline install/consul/istio.yaml]: I think we need "53:8600/udp" for Mac but "172.28.0.1:53:8600/udp" for linux, due to docker network differences.  @GregHanson can you confirm that?

File: samples/bookinfo/consul/bookinfo.yaml (hunk 1 above)
Comment (linsun) [reply samples/bookinfo/consul/bookinfo.yaml]: nice, thanks!!

File: install/consul/istio.yaml [hunk 4]
```diff
@@ -1,3 +1,5 @@
+# GENERATED FILE. Use with Docker-Compose and Consul
+# TO UPDATE, modify files in install/consul/templates and run install/updateVersion.sh
```
Comment (linsun) [inline install/consul/istio.yaml]: would you also put consul's istio.yaml in this install/consul/templates and rename it to istio.yaml.tmpl?  (I'm not seeing the change for that, thus the ask.)

Comment (linsun) [reply install/consul/istio.yaml]: i saw the missing file you just added, so this one is good :)

File: install/consul/istio.yaml (hunk 3 above)
Comment (rshriram) [reply install/consul/istio.yaml]: Yes, I spoke to @GregHanson . He is adding two YAMLs (one for mac and one for linux)

**Comment by kyessenov** [comment]
LGTM from release perspective

**Pull request merged**